### ISSUE TITLE
suJSON to CSV conversion

### ISSUE THREAD

**Issue by slhck** [LucjanJanowski/translator-to-suJSON#6]
For analyzing the data with Excel or R, you may want to go back from suJSON to a [tidy CSV](https://vita.had.co.nz/papers/tidy-data.pdf).

This would only be a view of the original data, of course, mostly ratings.

TODOs:

- [ ] Define the output format
- [ ] Write a conversion tool

**Comment by slhck** [comment]
@lukedrum Once #11 is merged I or we can work on the export functions.

**Comment by lukedrum** [comment]
Sure! We can do it. What kind of output format do we need? Tidy data format?

| ID | Subject | Score | PVS | SRC | HRC |
| --- | --- | --- | --- | --- | --- |
| 1 | 1 | src1_hrc1.jpg | 1 | 1 |
| 2 | 1 | src2_hrc1.jpg | 2 | 1 |

**Comment by slhck** [comment]
Yes, that would be one possible format! The ID is not needed though, I think (use `index=False` for `pd.write_csv`). I would start with that!
